Tests/Functional/Domain/Repository/Product/TeaRepositoryTest.php

```diff
@@ -133,4 +134,53 @@ public function addAndPersistAllCreatesNewRecord(): void
         self::assertIsArray($databaseRow);
         self::assertSame($title, $databaseRow['title']);
     }
+
+    /**
+     * @test
+     */
+    public function findByOwnerUidFindsEventWithTheGivenOwnerUid(): void
```

Comment:
I guess "Event" is a copy and paste or autocompletion error and should be "Tea" instead?

Reply:
I still see "Event" at https://github.com/TYPO3-Documentation/tea/pull/860/files#diff-516bb26cd1c2b4fe34e6ef45b6a15818f42846677642456d762045c06419fcc0R153, did I miss something? (I don't do reviews on GitHub that often) or did you miss it while pushing?

Reply:
Argh, sorry, my mistake. There were indeed multiple occurrences (probably from copy'n'pasting), and I only fixed one. Done and repushed.

Reply:
Okay, I'll try to remember and mention it next time :)